src/app/Menu/MenuPost.js

```diff
@@ -11,15 +12,30 @@ const MenuPost = ({
   reblog,
   isReblogged,
   canReblog,
-  openCommentingDraft,
   isPostLiked,
   isPostDisliked,
   likePost,
   unlikePost,
   dislikePost,
   content,
   onEdit,
+  modal
 }) => {
+  const handleReplyButtonClick = (e) => {
+    e.stopPropagation();
+    if (modal) {
+      animateScroll.scrollMore(400, { containerId: 'busyModal' });
```

Comment:
The scroll position should rely on comment form position and not a defined value. Actually when i click comment multiple time the scroll will keep increase to 400. Are we sure we want to add a new dependancy and complexity in the code for a smooth effect? We could use hashtag link instead and no worry about this.